components/fxa-client/src/http_client.rs

```diff
@@ -131,6 +137,19 @@ impl FxAClient for Client {
         self.make_oauth_token_request(config, body)
     }
 
+    fn oauth_introspection_with_refresh_token(  // TODO
+        &self,
+        config: &Config,
+        refresh_token: &str,
+        scopes: &[&str],
+    ) -> Result<IntrospectResponse> {
+        let body = json!({
+            "token_type_hint": "refresh_token",
+            "token": refresh_token
+        });
+        self.make_oauth_introspection_request(config, body)
```

Comment:
You can inline this helper function as it's only called once.

Reply:
I agree with @eoger, we should just call `Self::make_request` directly inside `oauth_introspection_with_refresh_token` here rather than calling out to a single-use helper. If and when we make another method that hits the same HTTP endpoint, we can factor it out into a helper function at that point.